examples/run/CMakeLists.txt

```diff
@@ -1,5 +1,16 @@
 set(TARGET llama-run)
 add_executable(${TARGET} run.cpp linenoise.cpp/linenoise.cpp)
+
+# TODO: avoid copying this code block from common/CMakeLists.txt
+set(LLAMA_RUN_EXTRA_LIBS "")
+if (LLAMA_CURL)
+    find_package(CURL REQUIRED)
+    target_compile_definitions(${TARGET} PUBLIC LLAMA_USE_CURL)
+    include_directories(${CURL_INCLUDE_DIRS})
+    find_library(CURL_LIBRARY curl REQUIRED)
+    set(LLAMA_RUN_EXTRA_LIBS ${LLAMA_RUN_EXTRA_LIBS} ${CURL_LIBRARY})
+endif ()
```

Comment:
@ericcurtin FYI, seems like `llama-run` was not linked against curl on windows, which will result in compilation error (cannot find `curl/curl.h` header). I made this quick fix which should work for now.

By default, only `libcommon` is linked against curl.

Reply:
Thanks @ngxson FWIW we are moving away from using llama-run in RamaLama, I wrote a kinda client server thing that automatically terminates both processes on completion instead (the experience is similar to llama-run but better). I was starting to dislike the duplication of efforts in llama-run and llama-server. So I want to try and consolidate on llama-server.

It's a neat tool to test llama-server with all kinds of models, I'm almost ready to push the full version of it to main in RamaLama.

I'll still be happy to look at PRs around llama-run, etc. but it probably won't be actively used in RamaLama anymore at least.